Bugzilla/Config/Elastic.pm

```diff
@@ -16,18 +16,23 @@ use Bugzilla::Config::Common;
 our $sortkey = 1550;
 
 sub get_param_list {
-  return (
-    {
-        name    => 'elasticsearch_nodes',
-        type    => 't',
-        default => 'localhost:9200',
-    },
-    {
-        name    => 'elasticsearch_index',
-        type    => 't',
-        default => 'bugzilla',
-    },
-  );
+    return (
+        {
+            name    => 'elasticsearch',
```

Comment:
What are these three parameters? Why is this one called simply 'elasticsearch'? Are these documented elsewhere?